Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(useAci: true)
```

Comment:
Why are you removing the `buildPlugin.recommendedConfigurations()`? I thought that was the correct way to execute the `buildPlugin` function

Reply:
https://github.com/jenkins-infra/pipeline-library/pull/145 and in particular it is impossible for someone without write access to the repo to accurately test changes to `jenkins.version` so long as the `Jenkinsfile` refers to other Jenkins versions.

Reply:
Thanks for the explanation!